text/0124-documents-api.md

```diff
@@ -53,7 +53,7 @@ Unique identifier of an index.
 | Field                    | Type                     | Required |
 |--------------------------|--------------------------|----------|
 | `offset`                 | Integer / `null`         | false    |
-| `limit`                  | String / `null`          | false    |
+| `limit`                  | Integer / `null`         | false    |
```

Comment:
huh, that's interesting. Why was the type string?

Reply:
It's a quick copy/paste jutsu that went through the reviews like a super ninja. 🥲